cypress/e2e/StaffPage.cy.tsx

```diff
@@ -0,0 +1,16 @@
+import { StaffPageTestFunctions } from 'cypress/utils/StaffPage/StaffPageTestFunctions';
+
+const StaffPage = new StaffPageTestFunctions();
+
+describe('StaffPage.cy.tsc', () => {
+  beforeEach(() => {
+    cy.visit('/staff');
+  });
+
+  it('should render the staff page', () => {
+    StaffPage.testStaffList();
+    StaffPage.testStaffLink();
+    StaffPage.testStaffDetailsPL();
+    StaffPage.testStaffDetailsEN();
+  });
```

Comment:
divide tests for english and polish version into separate test blocks, also testing links should be in separate test block. Testing Staff details should be in completely another file since you test a different page

Reply:
you're missing tests for header here as well